DOC-6342 reinstate log flags as aliases (#16619)
cockroachdb/docs: _includes/v23.1/misc/logging-flags.md

Flag | Description
-----|------------

> `--log` |  Configure logging parameters by specifying a YAML payload. For details, see [Configure logs](configure-logs.html#flag). If a YAML configuration is not specified, the [default configuration](configure-logs.html#default-logging-configuration) is used.<br/><br/>`--log-config-file` can also be used.<br/><br/>**Note:** The logging flags below cannot be combined with `--log`, and can be defined instead in the YAML payload.

Reviewer: ```suggestion
`--log` |  Configure logging parameters by specifying a YAML payload. For details, see [Configure logs](configure-logs.html#flag). If a YAML configuration is not specified, the [default configuration](configure-logs.html#default-logging-configuration) is used.<br/><br/>`--log-config-file` can also be used.<br/><br/>**Note:** The logging flags below cannot be combined with `--log`, but can be defined instead in the YAML payload.
```

> `--log-config-file` |  Configure logging parameters by specifying a path to a YAML file. For details, see [Configure logs](configure-logs.html#flag). If a YAML configuration is not specified, the [default configuration](configure-logs.html#default-logging-configuration) is used.<br/><br/>`--log` can also be used.<br/><br/>**Note:** The logging flags below cannot be combined with `--log-config-file`, and can be defined instead in the YAML file.

Reviewer: ```suggestion
`--log-config-file` |  Configure logging parameters by specifying a path to a YAML file. For details, see [Configure logs](configure-logs.html#flag). If a YAML configuration is not specified, the [default configuration](configure-logs.html#default-logging-configuration) is used.<br/><br/>`--log` can also be used.<br/><br/>**Note:** The logging flags below cannot be combined with `--log-config-file`, but can be defined instead in the YAML file.
```

`--log-dir` | An alias for the [`--log`](configure-logs.html#flag) flag, for configuring the log directory where log files are stored and written to. Specifically, `--log-dir=XXX` is an alias for `--log='file-defaults: {dir: XXX}'`.<br/><br/>Setting `--log-dir` to a blank directory (`--log-dir=""`) disables logging to files.

> `--log-group-max-size` | An alias for the [`--log`](configure-logs.html#flag) flag, for configuring the maximum logging group (i.e., `cockroach`, `cockroach-sql-audit`, `cockroach-auth`, `cockroach-sql-exec`, `cockroach-pebble`) size before deleting the oldest log file. Specifically, `--log-group-max-size=XXX` is an alias for `--log='file-defaults: {max-group-size: XXX}'`. Accepts a valid file size, such as `--log-group-max-size=1GiB`.<br/><br/>**Default:** `100MiB`

Reviewer: ```suggestion
`--log-group-max-size` | An alias for the [`--log`](configure-logs.html#flag) flag, for configuring the maximum size for a logging group (for example, `cockroach`, `cockroach-sql-audit`, `cockroach-auth`, `cockroach-sql-exec`, `cockroach-pebble`), after which the oldest log file is deleted. `--log-group-max-size=XXX` is an alias for `--log='file-defaults: {max-group-size: XXX}'`. Accepts a valid file size, such as `--log-group-max-size=1GiB`.<br/><br/>**Default:** `100MiB`
```

> `--log-file-max-size` | An alias for the [`--log`](configure-logs.html#flag) flag, for configuring the maximum size of log files. before a new log file is created. Specifically, `--log-file-max-size=XXX` is an alias for `--log='file-defaults: {max-file-size: XXX}'`. Accepts a valid file size, such as `--log-file-max-size=2MiB`. **Requires** logging to files.<br/><br/>**Default:** `10MiB`

Reviewer: ```suggestion
`--log-file-max-size` | An alias for [`--log`](configure-logs.html#flag), used to specify the maximum size that a log file can grow before a new log file is created. `--log-file-max-size=XXX` is an alias for `--log='file-defaults: {max-file-size: XXX}'`. Accepts a valid file size, such as `--log-file-max-size=2MiB`. **Requires** logging to files.<br/><br/>**Default:** `10MiB`
```

> `--log-file-verbosity` | An alias for the [`--log`](configure-logs.html#flag) flag, for configuring log output [severity level](logging.html#logging-levels-severities). Specifically, `--log-file-verbosity=XXX` is an alias for `--log='file-defaults: {filter: XXX}'`. When a severity is specified, such as `--log-file-verbosity=WARNING`, only log messages which are at or above the specified severity level are written to the target log file. **Requires** logging to files.<br/><br/>**Default:** `INFO`

Reviewer: ```suggestion
`--log-file-verbosity` | An alias for [`--log`](configure-logs.html#flag), used to specify the minimum [severity level](logging.html#logging-levels-severities) of messages that are logged. `--log-file-verbosity=XXX` is an alias for `--log='file-defaults: {filter: XXX}'`. When a severity is specified, such as `--log-file-verbosity=WARNING`, log messages that are below the specified severity level are not written to the target log file. **Requires** logging to files.<br/><br/>**Default:** `INFO`
```

> `--logtostderr` |  An alias for the [`--log`](configure-logs.html#flag) flag, for configuring log output to the `stderr` sink. Specifically, `--logtostderr=XXX` is an alias for `--log='sinks: {stderr: {filter: XXX}}'`. Accepts a valid [severity level](logging.html#logging-levels-severities). If no value is specified, the default value for the command is inferred as follows: `INFO` for server commands, `WARNING` for client commands.<br/><br/>Setting `--logtostderr=NONE` disables logging to `stderr`.<br><br>**Default:** `UNKNOWN`

Reviewer: ```suggestion
`--logtostderr` |  An alias for [`--log`](configure-logs.html#flag), to optionally output log messages at or above the configured  [severity level](logging.html#logging-levels-severities) to the `stderr` sink. `--logtostderr=XXX` is an alias for `--log='sinks: {stderr: {filter: XXX}}'`. Accepts a valid [severity level](logging.html#logging-levels-severities). If no value is specified, by default messages related to server commands are logged to `stderr` at `INFO` severity and above, and messages related to client commands are logged to `stderr` at `WARNING` severity and above.<br/><br/>Setting `--logtostderr=NONE` disables logging to `stderr`.<br><br>**Default:** `UNKNOWN`
```

> `--no-color` | An alias for the [`--log`](configure-logs.html#flag) flag, for configuring colorization of log output to the `stderr` sink. Specifically, `--no-color=XXX` is an alias for `--log='sinks: {stderr: {no-color: XXX}}'`. Accepts either `true` or `false`.<br/><br/>When set to `false`, messages logged to `stderr` are colorized based on [severity level](logging.html#logging-levels-severities).<br><br>**Default:** `false`

Reviewer: ```suggestion
`--no-color` | An alias for [`--log`](configure-logs.html#flag) flag, used to control whether log output to the `stderr` sinc is colorized. `--no-color=XXX` is an alias for `--log='sinks: {stderr: {no-color: XXX}}'`. Accepts either `true` or `false`.<br/><br/>When set to `false`, messages logged to `stderr` are colorized based on [severity level](logging.html#logging-levels-severities).<br><br>**Default:** `false`
```

> `--redactable-logs` | An alias for the [`--log`](configure-logs.html#flag) flag, for configuring [redaction markers](configure-logs.html#redact-logs) in log messages. Specifically, `--redactable-logs=XXX` is an alias for `--log='file-defaults: {redactable: XXX}'`. Accepts `true` or `flase`.<br><br>**Default:** `false`

Reviewer: ```suggestion
`--redactable-logs` | An alias for [`--log`](configure-logs.html#flag) flag, used to whether [redaction markers](configure-logs.html#redact-logs) are used in place of secret or sensitive information in log messages. `--redactable-logs=XXX` is an alias for `--log='file-defaults: {redactable: XXX}'`. Accepts `true` or `false`.<br><br>**Default:** `false`
```

> `--sql-audit-dir` | An alias for the [`--log`](configure-logs.html#flag) flag, for configuring log output of the `SENSITIVE_ACCESS` [logging channel](logging-overview.html#logging-channels) to the indicated directory. Specifically, `--sql-audit-dir=XXX` is an alias for `--log='sinks: {file-groups: {sql-audit: {channels: SENSITIVE_ACCESS, dir: ...}}}'`.<br><br>Note that enabling `SENSITIVE_ACCESS` logs can negatively impact performance. As a result, we recommend using the `SENSITIVE_ACCESS` channel for security purposes only. For more information, see [Logging use cases](logging-use-cases.html#security-and-audit-monitoring).

Reviewer: ```suggestion
`--sql-audit-dir` | An alias for [`--log`](configure-logs.html#flag), used to optionally confine log output of the `SENSITIVE_ACCESS` [logging channel](logging-overview.html#logging-channels) to a separate directory. `--sql-audit-dir=XXX` is an alias for `--log='sinks: {file-groups: {sql-audit: {channels: SENSITIVE_ACCESS, dir: ...}}}'`.<br><br>Enabling `SENSITIVE_ACCESS` logs can negatively impact performance. As a result, we recommend using the `SENSITIVE_ACCESS` channel for security purposes only. For more information, refer to [Security and Audit Monitoring](logging-use-cases.html#security-and-audit-monitoring).
```
